xds/internal/env/env.go

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package env acts a single source of definition for all environment variables
+// related to the xDS implementation in gRPC.
+package env
+
+import (
+	"os"
+	"strings"
+)
+
+const (
```

Comment:
```go
const (
	prefix               = "GRPC_XDS_"
	bootstrapFileNameEnv = prefix + "BOOTSTRAP"
	xdsV3SupportEnv      = prefix + "EXPERIMENTAL_V3_SUPPORT"
)
```

Reply:
I specifically didn't want this format since I want us to be able to easily grep for the full name of the environment variable in the code base. I was planning to fix the `envconfig` package also to do the same. This pattern is also mentioned in one of the style guides.

Reply:
Sounds good!